lix/packages/sdk/src/newLix.ts

```diff
@@ -19,11 +20,16 @@ export async function newLixFile(): Promise<Blob> {
 
 	try {
 		await sql`
+      CREATE TABLE ref (
+        name TEXT PRIMARY KEY,
+        commit_id TEXT
+      );
+
```

Comment:
Unsure what ref is for. Ignoring it for now to get you into the monorepo

Reply:
@samuelstroschein    ok!   ref is needed to know where we are, using timestamps is not possible in a production environment for distributed systems the use of timestamp until now was only a prototype hack. ping me if you want to discuss this more.

Reply:
I thought we are going to use the `parent_id` property to derive where we are from?